### ISSUE TITLE
slang.tcl: json api

### ISSUE THREAD

**Issue by skydrome** [horgh/eggdrop-scripts#11]
On the chance you still update these,
UD has an api http://api.urbandictionary.com/v0/define?term=
is that easier to parse? probably faster to download too as its less data

**Comment by horgh** [comment]
Yeah, that would probably be better. I'm not sure that existed when I created this. Maybe if the parsing breaks badly enough one day I will convert over to it. I probably won't have time otherwise :-). I don't see documentation for it on their site though strangely.
